lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -83,18 +83,122 @@ describe('NxCombobox', function() {
         expect(inputElement).toHaveValue('Foo');
       });
 
-  it('sets aria-expanded on the input to true when focused',
+  it('sets aria-expanded to true on the input when initially focused and there are matches', async function() {
+    const { getByRole } = quickRender({ matches: [{ id: '1', displayName: 'Foo' }] }),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    // initial focus of the input
+    inputElement.focus();
+    expect(inputElement).toHaveAttribute('aria-expanded', 'true');
+
+  // visibility of dropdown when focused is tested in visual tests
+  });
+
+  it('set aria-expanded to false on the input if there are no matches, even if focused', async function() {
+    const { rerender, getByRole } = quickRender(),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    inputElement.focus();
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    rerender(<NxCombobox {...minimalProps} value= {'foo'}/>);
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    rerender(<NxCombobox {...minimalProps} loading={true}/>);
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
+
+    rerender(<NxCombobox { ...minimalProps } loadError={'boo'} />);
+    expect(inputElement).toHaveAttribute('aria-expanded', 'false');
```

Comment:
Also test the cases where loading and error would result in `aria-expanded` being true: when `loading` or `loadError` are set and `value` is also set

Reply:
> Also test the cases where loading and error would result in aria-expanded being true: when loading or loadError are set and value is also set

I was wrong about this per https://github.com/sonatype/sonatype-react-shared-components/pull/876#discussion_r1063501298